functions/v2/response-streaming/src/main/java/com/google/StreamBigQuery.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google;
+
+import com.google.cloud.bigquery.BigQuery;
```

Comment:
Can we add region tags surrounding the sample so we can pull them into docs?
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/5a8d34d0eb014a10aff5347450c3b70f325d448c/functions/v2/pubsub/src/main/java/functions/SubscribeToTopic.java#L19